spark/pom.xml

```diff
@@ -731,6 +726,77 @@
       </dependencies>
     </profile>
 
+    <profile>
+      <id>pyspark</id>
```

Comment:
I'd suggest calling this pyspark_local or something - for those who have pyspark already in their cluster, this should not be necessary?